src/server/authMiddleware.ts

```diff
@@ -72,6 +93,13 @@ const createAuthMiddleware =
 	async (req: NextRequest) => {
 		console.debug('Auth middleware starts');
 
+		if (options.publicRoutes && options.privateRoutes) {
+			console.warn(
+				'Both publicRoutes and privateRoutes are defined. Ignoring privateRoutes.'
+			);
+			options.privateRoutes = undefined;
```

Comment:
Assignment of arguments are bad practice, avoid doing so and just handle in the function